### PR TITLE
FEAT: add syntax highlighting in Code Generator Modal

### PR DESCRIPTION
**Pull request by iamtabrezkhan**
Hi @liyasthomas, as discussed in #1507 issue I have added syntax highlighting in generated code modal.

I have used SmartAceEditor as you suggested.
Added a `language` property in codegen files based on what ace editor supports.

syntax-highlighting

PS: I am new to vuejs.

### CODE REVIEW

**Comment by liyasthomas** [comment]
@iamtabrezkhan thanks for this quick PR. I'll look into this as soon as possible. ✨

**Review by liyasthomas — APPROVED** [review]
LGTM 🚀

File: components/http/CodegenModal.vue [hunk 1]
```diff
@@ -46,15 +46,21 @@
...
+        :options="{
+          maxLines: '10',
+          minLines: '10',
+          fontSize: '16px',
+          autoScrollEditorIntoView: true,
+          readOnly: true,
+          showPrintMargin: false,
+          useWorker: false,
+        }"
         ref="generatedCode"
```
Comment (liyasthomas) [inline components/http/CodegenModal.vue:62]:
```suggestion
        styles="rounded-b-lg"
        ref="generatedCode"
```

**Review by AndrewBastin — APPROVED** [review]
Looks good to me!

**Pull request merged**